backend/database/models.py

```diff
@@ -0,0 +1,26 @@
+from .db import db
+
+# Create a class for all table that should be in the database
+# Should have:
+# User, picture,
+
+
+class User(db.Document):
+    email = db.EmailField(required=True, unique=True)
+    username = db.StringField(required=True, unique=True)
+    fname = db.StringField(required=True)
+    lname = db.StringField()
+    password = db.StringField(required=True)
+    own_pictures = db.ListField()
+    liked_pictures = db.ListField()
```

Comment:
This should go in the picture schema

Reply:
I changed a bit what is here. Let's talk about that at the next meeting